docker/test_env/non_dev_besu.go

```diff
@@ -28,7 +28,7 @@ import (
 )
 
 const (
-	BESU_IMAGE = "hyperledger/besu"
+	defaultNonDevBesuImage = "hyperledger/besu:24.1"
```

Comment:
I wonder if we are using this nondev Besu anywhere... as it might be totally broken. If I remember correctly the code was adjusted to PoW Besu and I never updated it. I think we should be using older Besu image here.